### PR TITLE
Add Documentation

### PR DESCRIPTION
**Pull request by prg3**
First kick at dokumentation.. will need some tweaks over time as the FAQ fills up

### CODE REVIEW

**Comment by prg3** [comment]
Removal of the _autosave from KiCAD wasn't intentional, however seems reasonable that an autosave file doesn't need to be here? Might be worth a .gitignore to protect against weird commits

File: Klipper_Expander/Documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    - [Why do I want this?](#why-do-i-want-this)
+    - [How do I get one?](#how-do-i-get-one)
+
+<!-- /TOC -->
+
+# Basic
+
+First we will go through the basic functionality of the board that most users will be looking for
+
+## Indicator LEDs
```
Comment (timmit99) [inline Klipper_Expander/Documentation/README.md:32]: Here i would add labels to each LED (D1,D2...) and add a 4th section for MOSFET Status LED's
- Board Ready
- USB Power (5V)
- Vin Power
- MOSFET Status

File: Klipper_Expander/Documentation/README.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+```
+[neopixel Pixel]
+pin: PB1
+chain_count: 1
+initial_RED: 0.9
+initial_GREEN: 0.3
+initial_BLUE: 0.0
+```
+
+# Setup
```
Comment (timmit99) [inline Klipper_Expander/Documentation/README.md:135]: @john-- thoughts on having the setup guide inside here vs a separate guide document?
I see benefits to either direction

File: Klipper_Expander/Documentation/README.md (hunk 1 above)
Comment (prg3) [reply Klipper_Expander/Documentation/README.md:32]: Done!

File: Klipper_Expander/Documentation/README.md (hunk 2 above)
Comment (prg3) [reply Klipper_Expander/Documentation/README.md:135]: I think if we break it apart, we break it apart into a bunch of smaller docs, but it doesn't have to be.

File: Klipper_Expander/Documentation/README.md (hunk 1 above)
Comment (timmit99) [reply Klipper_Expander/Documentation/README.md:32]: Looks good!

File: Klipper_Expander/Documentation/README.md (hunk 2 above)
Comment (timmit99) [reply Klipper_Expander/Documentation/README.md:135]: I'm ok with it being in one doc for now, we can see if it grows and needs to be split.

**Pull request merged**